contentcuration/contentcuration/utils/publish.py

```diff
@@ -454,6 +454,7 @@ def write_to_zipfile(filename, content, zf):
 
 
 def write_assessment_item(assessment_item, zf):  # noqa C901
+    channel_id = assessment_item.get_channel_id()
```

Comment:
I don't see this method defined on the AssessmentItem class, I think we might need to pass this in from the outside too, from the contentnode.

Reply:
Thanks for catching this, I made the change